src/sqlfluff/dialects/dialect_tsql.py

```diff
@@ -2202,6 +2203,20 @@ class TableIndexClause(BaseSegment):
                 "COLUMNSTORE",
                 "INDEX",
             ),
+            Sequence(
```

Comment:
```
CLUSTERED COLUMNSTORE INDEX ORDER (column [,...n])
```
Looking at the docs, this is also an option so we could probably add this now?

Think we'd just need to update the lines above this to:
```python
Sequence(
    "CLUSTERED",
    "COLUMNSTORE",
    "INDEX",
    Sequence(
        "ORDER",
        Bracketed(
            Delimited(
                Ref("ColumnReferenceSegment")
            ),
        ),
        optional=True
    ),
),
```

Reply:
@jpy-git good catch!  I'll update.